src/client/components/BookingForms/AddMeal.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import "./forms.css";
+import { useForm } from "react-hook-form";
+import { joiResolver } from "@hookform/resolvers/joi";
+import Joi from "joi";
+
+const mealsValidSchema = Joi.object({
```

Comment:
Technically it would be possible to use the same schema definition in the frontend and backend, which is what you really want since now you risk changing the rules on one side but not the other.